templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -6,142 +6,341 @@
-   ],
    "source": [
-    "# To get help on the CLI\n",
-    "!llmforge anyscale finetune --help"
+    "## Examples\n",
+    "\n",
+    "Here are some examples for default mode and custom mode:\n",
+    "\n",
+    "### Default Mode"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As the command runs, you can monitor a number of built-in metrics in the `Metrics` tab under `Ray Dashboard`, such as the number of GPU nodes and GPU utilization.\n",
     "\n",
-    "<img src=\"https://raw.githubusercontent.com/anyscale/templates/main/templates/fine-tune-llm_v2/assets/gpu-usage.png\" width=500px/>\n",
+    "--------- \n",
+    "**Task:** \n",
+    "\n",
+    "Fine-tune llama-3-8b-instruct in default mode (LoRA rank 8). Just giving the dataset.\n",
+    "\n",
+    "**Command:**\n",
+    "```bash\n",
+    "llmforge anyscale finetune training_configs/default/meta-llama/Meta-Llama-3-8B-Instruct-simple.yaml --default\n",
+    "```\n",
+    "\n",
+    "**Config:**\n",
     "\n",
-    "Depending on whether you are running LoRA or full-param fine-tuning, you can continue with step 2(a) or step 2(b). To learn more about LoRA vs. full-parameter, see the cookbooks."
+    "```yaml\n",
+    "model_id: meta-llama/Meta-Llama-3-8B-Instruct\n",
+    "train_path: s3://...\n",
+    "```"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
+    "--------- \n",
     "\n",
-    "## Step 2(a) - Serving the LoRA fine-tuned model\n",
+    "**Task:** \n",
     "\n",
-    "Upon the job completion, you can see the LoRA weight storage location and model ID in the log, such as the one below:\n",
+    "Fine-tune llama-3-8b-instruct in default mode but also control parameters like `learning_rate` and `num_epochs`. \n",
     "\n",
-    "```shell\n",
-    "Note: LoRA weights will also be stored in path {ANYSCALE_ARTIFACT_STORAGE}/lora_fine_tuning under meta-llama/Llama-2-8b-chat-hf:sql:12345 bucket.\n",
+    "**Command:**\n",
+    "```bash\n",
+    "llmforge anyscale finetune training_configs/default/meta-llama/Meta-Llama-3-8B-Instruct-custom.yaml --default\n",
     "```\n",
     "\n",
-    "You can specify this URI as the dynamic_lora_loading_path [docs](https://docs.anyscale.com/examples/deploy-llms#more-guides) in the llm serving template, and then query the endpoint.\n",
+    "**Config:**\n",
+    "\n",
+    "```yaml\n",
+    "model_id: meta-llama/Meta-Llama-3-8B-Instruct\n",
+    "train_path: s3://...\n",
+    "valid_path: s3://...\n",
+    "num_epochs: 3\n",
+    "learning_rate: 1e-4         \n",
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Custom"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---------\n",
+    "**Task:** \n",
+    "\n",
+    "Fine-tune llama-3-8b-instruct in custom mode (model is supported in default mode) on 16xA10s (auto mode uses 8xA100-80G) with context length of 512.\n",
```

Comment:
Nit: "Fine-tune llama-3-8b-instruct in custom mode (model is supported in default mode)" for a place like this the `model is supported in default mode` phrase is wordy and a bit confusing. A shortname like "core models" or something is useful.